example/spec/parallel/lzardsApiClientTest/LzardsApiClientSpec.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
```

Comment:
I think we should combine example/spec/parallel/lzardsBackup/LzardsBackupSpec.js with this one, and keep the test in lzardsBackup directory

Reply:
Then we don't need the added testGranule.dat and testGranule.jpg, and one less test to worry about!

Reply:
combined. excellent suggestion